Add BBE for `Regex` replace operations (#5679)
ballerina-platform/ballerina-distribution: examples/regexp-replace-operations/regexp_replace_operations.md

# RegExp operations

> The ``RegExp`` type supports a set of anguage library functions for replacing patterns in strings.

Reviewer: ```suggestion
The `RegExp` type supports a set of language library functions to replace parts of strings that match specific patterns.
```
